### DISCUSSION TITLE
How to install qt6 with qml feature?

### DISCUSSION THREAD

**Discussion by rhcher** [microsoft/vcpkg#32833] [General]
I have a c++ qt source to build. There is a vcpkg.json file:
```json
{
    "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
    "dependencies": [
        "catch2",
        "fmt",
        {"name": "qtbase", "features": ["default-features"]},
        {"name": "qt5-declarative", "features": ["d3d12"]},
        {"name": "qtmultimedia", "features": ["qml"]},
        {"name": "qt5compat", "features": ["qml"]},
        {"name": "qttools", "features": ["qml"]},
        {"name": "qtactiveqt", "features": ["qml"]},
        "fontconfig",
        "freetype",
        "harfbuzz",
        "range-v3",
        "yaml-cpp"
    ]
}
```
and after vcpkg install and I want to build, the error appears:
![image](https://github.com/microsoft/vcpkg/assets/50723480/ab25a24f-cb43-4a81-aa87-b97696c076b7)
But why? I google everywhere and nobody have same issue with me.

**Comment by dg0yt** [comment]
`qt5-declarative` is from Qt 5, not Qt 6.

**Reply by rhcher** [reply]
Thanks, I know it, and remove it also doesn't make it work

**Comment by dg0yt** [comment]
You are not properly using vcpkg (integration). `vcpkg/packages` is a staging directory, not a permanent location. In case of CMake, reconfigure the project from a fresh build directory, pointing `CMAKE_TOOLCHAIN_FILE` to `scripts/buildsystems/vcpkg.cmake` during the very first run of CMake.
